### ISSUE TITLE
Definition of alternate symbols for events

### ISSUE THREAD

**Issue by scottb5612** [T-F-S/genealogytree#46]
Is it possible to redefine the symbols for death, birth etc ?
Or do I have to install a private version of  the entire geneaologytree package 
with the desired definitions?

**Comment by T-F-S** [comment]
Yes, you can change the symbols. I assume that you use *database processing* (chapter 7). The symbol printing is documented in *7.9 Formatting Events*.

The *event prefix* is the symbol. For example, `event prefix/birth` defaults to `\gtrsymBorn`. With

```latex
\gtrset{
  event prefix/birth=.....,
  event prefix/marriage=...,
}
```

you can switch to your own symbols (or text or whatever).

**Comment by scottb5612** [comment]
I wasn't using database processing...
I hadn't figured out how to suppress events in the output,
so that I could just generate a tree with names only, no other information inside the box.
Is there a way to do that (or do you want me to submit another issue?)

Thank you for your quick reply.
SBB

**Comment by T-F-S** [comment]
I still think that you use *database processing*, because otherwise you would create the content of each node manually... So, my next guess it that you use a template based on *database processing*.
Here, you could add `database format=name` to the option list to display only names.

For further help, a minimal working example (MWE) would be required to avoid guessing from my side.

**Comment by scottb5612** [comment]
That's a good point. I'll try database processing.
I have another issue, that has to do with drawing.. Will post separately
